crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -321,25 +321,20 @@ impl RealRibosome {
             .map(|(zome_name, def)| {
                 let mut dependencies = Vec::new();
 
-                if integrity_zomes.len() == 1 {
```

Comment:
This shortcut here may be a small gain in efficiency but it has the subtle side-effect that I can forget to specify a zome dependency in the dna manifest and get away with it if there is only one integrity zome in my dna. Arguably that might be okay behavior but it won't work anymore as soon as there are more than 1 integrity zomes and that difference in error behavior (or absence thereof) seems inconsistent and unexpected to me.

Reply:
I'd rather have it error consistently but have a decent error message that explains what the problem likely is. Which is what this PR does.